### DISCUSSION TITLE
WIKIJS CUSTOM

### DISCUSSION THREAD

**Discussion by Yoann-Paquet** [requarks/wiki#4935] [Help / Questions]
Bonjour je cherche à change la couleur de la barre de WikiJS  J'imagine que je dois utiliser du code css mais je n'arrive pas à faire le changement. Vous pouvez me donner la ligne de code à rentrer s'il vous plait?

![Capture](https://user-images.githubusercontent.com/86796797/151005046-de92005d-0cb8-4bf8-9beb-8c8899475a40.PNG)

**Comment by Kaphalor** [comment]
Didn't tried in in production, but according to the devtools, just change #000 to something different which fits your wish
```
.v-application .black {
    background-color: #000 !important;
    border-color: #000!important;
}
```
